### PR TITLE
OSDOCS-10526 updated billing into module

### PR DESCRIPTION
**Pull request by xJustin**
Updated the billing module and made sure it was in the right places. 

Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-10526
Link to docs preview:

https://77443--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-definition.html

https://77443--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html

https://77443--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_architecture/rosa-understanding.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by arendej — APPROVED** [review]
lgtm

**Comment by xueli181114** [comment]
LGTM

File: modules/rosa-sdpolicy-am-billing.adoc [hunk 1]
```diff
@@ -4,8 +4,11 @@
 // * rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc
 :_mod-docs-content-type: CONCEPT
 [id="rosa-sdpolicy-billing_{context}"]
-= Billing
+= Billing and pricing
 
-{product-title} is billed through Amazon Web Services (AWS) based on the usage of AWS components used by the service, such as load balancers, storage, EC2 instances, other components, and Red{nbsp}Hat subscriptions for the OpenShift service.
+ROSA is billed directly to your AWS account. ROSA pricing can be consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
```
Comment (opayne1) [inline modules/rosa-sdpolicy-am-billing.adoc:9]:
```suggestion
{product-rosa} is billed directly to your {aws-first} account. ROSA pricing can be consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
```

Comment (opayne1) [inline modules/rosa-sdpolicy-am-billing.adoc:9]: Just a small wording suggestion
```suggestion
{product-rosa} is billed directly to your {aws-first} account. ROSA pricing is based on an annual commitment or three-year commitment. The total cost of ROSA consists of two components:
```

Comment (opayne1) [reply modules/rosa-sdpolicy-am-billing.adoc:9]: If you choose to keep it the way it is I would change "can be" to "is" is it is based on the commitment you choose.
```suggestion
product-rosa} is billed directly to your {aws-first} account. ROSA pricing is consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
```

File: modules/rosa-sdpolicy-am-billing.adoc [hunk 2]
```diff
@@ -4,8 +4,11 @@
...
 
-Any additional Red{nbsp}Hat software must be purchased separately.
\ No newline at end of file
+* ROSA service fees
+* AWS infrastructure fees
+
+Visit the link:https://aws.amazon.com/rosa/pricing/[AWS pricing page] for more details.
```
Comment (opayne1) [inline modules/rosa-sdpolicy-am-billing.adoc:14]:
```suggestion
Visit link:https://aws.amazon.com/rosa/pricing/[{product-rosa} Pricing] for more details.
```

**Review by opayne1 — COMMENTED** [review]
My main comment is to replace product names with attributes, but I did leave a couple wording suggestions for you to consider. Otherwise looks great!

**Comment by xJustin** [comment]
Thanks @opayne1 ! I incorporated your suggestions.

File: modules/rosa-sdpolicy-am-billing.adoc [hunk 3]
```diff
@@ -4,8 +4,11 @@
...
 
-{product-title} is billed through Amazon Web Services (AWS) based on the usage of AWS components used by the service, such as load balancers, storage, EC2 instances, other components, and Red{nbsp}Hat subscriptions for the OpenShift service.
+{product-title} is billed directly to your {AWS} account. ROSA pricing is consumption based, with annual commitments or three-year commitments for greater discounting. The total cost of ROSA consists of two components:
 
-Any additional Red{nbsp}Hat software must be purchased separately.
\ No newline at end of file
+* ROSA service fees
+* AWS infrastructure fees
+
+Visit link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] page on the AWS website for more details.
```
Comment (skopacz1) [inline modules/rosa-sdpolicy-am-billing.adoc:14]: I would either add a "the" in front of the link or remove "page" after it, right now it sounds like it's missing something:
```suggestion
Visit the link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] page on the AWS website for more details.
```
```suggestion
Visit link:https://aws.amazon.com/rosa/pricing/[{product-title} Pricing] on the AWS website for more details.
```

**Review by skopacz1 — COMMENTED** [review]
Just one nit, let me know once it's addressed and I can merge

**Pull request merged**